apps/src/templates/instructions/InstructionsCsfLeftCol.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import classNames from 'classnames';
+import {connect} from 'react-redux';
+import PromptIcon from './PromptIcon';
+import HintDisplayLightbulb from '../HintDisplayLightbulb';
+import {getOuterHeight} from './utils';
+import commonStyles from '../../commonStyles';
+
+const PROMPT_ICON_WIDTH = 60; // 50 + 10 for padding
+const AUTHORED_HINTS_EXTRA_WIDTH = 30; // 40 px, but 10 overlap with prompt icon
+
+const styles = {
+  // bubble has pointer cursor by default. override that if no hints
+  noAuthoredHints: {
+    cursor: 'default',
+    marginBottom: 0
+  },
+  authoredHints: {
+    // raise by 20 so that the lightbulb "floats" without causing the original
```

Comment:
Not sure I understand how this comment relates to `marginBottom: 0`?

Reply:
Good catch. This seems like it might be a super old comment that's not relevant anymore. I'll clean up!